src/templates/index.hbs

```diff
@@ -62,10 +62,22 @@
                         <a class="page-scroll" href="#parceiro">Parceiros</a>
                     </li>
                     <li>
+
                         <a href="https://www.meetup.com/opensanca/" target="_blank">Eventos</a>
                     </li>
                     <li>
                         <a href="https://github.com/opensanca" target="_blank">Projetos</a>
+
+                        <a class="page-scroll" href="#apoie">Apoie</a>
```

Comment:
It seems like this anchor element is duplicated, can you confirm if that's correct, please?

Reply:
I change this file and fix this